### PR TITLE
Add appropriate empty message in notifications fragment when not logged in

### LINKED ISSUE #2369: Add appropriate message in notifications fragment when not logged in
**Actual Behaviour**

Empty view shows "No Notifications" in notification fragment when not logged in.

**Expected Behaviour**

Empty view should show appropriate message in notification fragment when not logged in.

**Screenshots of the issue**

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by ParthivMangukiya**
Fixes #2369

Changes: Added appropriate message "Please Login to see Notifications" in Notification Fragment when not logged in.

Screenshots for the change:

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/core/notifications/NotificationsFragment.java [hunk 1]
```diff
@@ -45,6 +45,14 @@ public static NotificationsFragment getInstance() {
         return new NotificationsFragment();
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        if (!AuthUtil.isUserLoggedIn()) {
+            noNotificationView.setText(R.string.login_to_see_notifications);
+        }
+    }
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/core/notifications/NotificationsFragment.java]: Should be added to onCreate, we don't want to branch out lifecycles

**Comment by ParthivMangukiya** [comment]
@iamareebjamal i have moved setText in onCreateView method. Please Review.

**Comment by ParthivMangukiya** [comment]
@iamareebjamal Should I make this change in current PR or #2372 ? Please review this so that I can send appropriate PR for #2372.

**Comment by iamareebjamal** [comment]
Send a new PR please. Also use a common string, login to continue

**Comment by ParthivMangukiya** [comment]
@iamareebjamal As I have commit this changes in #2378 PR, this PR and Issue is no longer needed.
Can I close this issue and PR?

**Comment by iamareebjamal** [comment]
Yes

**Pull request closed without merging**